src/components/CompareResults/RevisionRow.tsx

```diff
@@ -181,7 +181,7 @@ function RevisionRow(props: RevisionRowProps) {
           <div className='graph-link-button-container'>
             <IconButton aria-label='graph link' size='small'>
               <Link href={graphLink} target='_blank'>
```

Comment:
I realize this now, but it's super weird to have a Link _inside_ a Button. Something to be looked at later...